### DISCUSSION TITLE
Building a standalone binary

### DISCUSSION THREAD

**Discussion by MrIkso** [termux/termux-packages#14612] [Q&A]
Hello im trying build standalone a ninja and cmake via the termux-packages build system. Its work, but I'm i don't know how to build binaries with dependencies, without linking to prefix folder . Reason - use binary outside termux app.

**Comment by SunPodder** [comment]
That's called static building

**Reply by MrIkso** [reply]
I decided this case without termux-packages build system. Used my build script and ndk
